Sources/App/Utilities/SemverVersion.swift

```diff
@@ -86,24 +107,29 @@ struct SemverVersion {
     func next(_ bump: Bump) -> SemverVersion {
         var version = self
         if var prerelease = version.prerelease {
+            let p: Prerelease?
```

Comment:
Why the local var?

Reply:
The previous one is not optional, and it can be set to `nil` now.